### ISSUE TITLE
[Request] Add a header and some sort of "I'm a bot" footer message to each comment

### ISSUE THREAD

**Issue by leftbones** [thatjpcsguy/squabbles-summarybot#2]
I realized I've been seeing summarybot everywhere without even knowing it, because it's not clearly marked as a bot by anything but the name.

I'd recommend something like this:

```
I tried to summarize this article:

[article summary goes here]

(This action was performed by a bot)
```

Or something like that. I know it's not required, but it seemed like bots on Reddit did it as more of a courtesy.

**Comment by etusm** [comment]
Agreed, this would be an easy add and would be nice to have. Maybe link back to this repo as well.

**Comment by thatjpcsguy** [comment]
Funnily enough I was just looking at solving this today. I'm trying to find a way of it being informative but also not too intrusive or repetitive. Most bots I've seen on reddit use some sort of markdown formatting for smaller text underneath the comment, but squabblr doesnt seem to support that.

**Comment by etusm** [comment]
I think, maybe, leave the markdown and links and stuff for the bot's actual profile page since that seems like a good place to put it and maybe also do a "self mention" to the bots profile. 

Something like "(This summary was performed by /u/summarybot)" since that self mention should auto link to the bot's profile.
